### PR TITLE
Asertion Fix 

### PR DESCRIPTION
**Pull request by daschuer**
This fixes a assertion.

### CODE REVIEW

**Comment by daschuer** [comment]
@Be-ing: Here as requested.

File: src/library/autodj/autodjprocessor.cpp [hunk 1]
```diff
@@ -617,7 +617,7 @@ void AutoDJProcessor::playerPositionChanged(DeckAttributes* pAttributes,
     }
 
     if (m_eState == ADJ_IDLE) {
-        if (!thisDeckPlaying && thisPlayPosition < 1) {
+        if (!thisDeckPlaying) {
```
Comment (Be-ing) [inline src/library/autodj/autodjprocessor.cpp:620]: Why did you revert this? Doesn't this break Full Track mode with a transition time of 0?

Comment (Be-ing) [reply src/library/autodj/autodjprocessor.cpp:620]: Oh I think this makes sense with the other changes... I have to think about this whole set of changes in more detail...

Comment (Be-ing) [reply src/library/autodj/autodjprocessor.cpp:620]: I tested and Full Track mode with a transition time of 0 is working with this PR.

**Pull request merged**